src/app/characters/characters.service.ts

```diff
@@ -7,10 +7,14 @@ import { environment } from '../../environments/environment';
 import { Character } from '../core/character.model';
 import { ContextService } from '../core/core.service';
 
-interface MarvelResponse {
+export interface MarvelResponse {
   attributionText: string;
   data: {
```

Comment:
I think that we only need the `total` property from `data`

Reply:
Done.